docs/docs/api/gestures/base-gesture-callbacks.md

```diff
@@ -12,6 +12,10 @@ Set the `onStart` callback.
 
 Set the `onEnd` callback.
 
+### `onFinalize(callback)`
+
+Set the `onFinalize` callback.
```

Comment:
Can we add a bit more explanation on what each callback does? Not only this one but also the remaining ones. e.g.:
onBegan – set the callback that is being called when given gesture handler starts receiving touches. At the moment of this callback the handler is not yet in an active state and we don't know yet if it will recognize the gesture at all.

Reply:
Done in https://github.com/software-mansion/react-native-gesture-handler/pull/1749/commits/dfec565565fc84f49b3f49cef4fcf442f7869048